cmd/kube-proxy/app/server.go

```diff
@@ -115,7 +117,8 @@ func NewProxyConfig() *ProxyServerConfig {
 		HealthzBindAddress: net.ParseIP("127.0.0.1"),
 		OOMScoreAdj:        qos.KubeProxyOOMScoreAdj,
 		ResourceContainer:  "/kube-proxy",
-		SyncPeriod:         30 * time.Second,
+		IptablesSyncPeriod: 30 * time.Second,
+		ConfigResyncPeriod: 10 * time.Minute,
```

Comment:
Renamed again.  I bumped to 15 minutes by default - I'm hesitant to go
larger unless you're sure it's the right default.

On Thu, Oct 15, 2015 at 8:51 AM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> I opened an issue to make list watch jitter in general - I think we should
> do it for every set of calls.
>
> On Oct 15, 2015, at 8:49 AM, Wojciech Tyczynski <notifications@github.com>
> wrote:
>
> In cmd/kube-proxy/app/server.go
> <https://github.com/kubernetes/kubernetes/pull/15682#discussion_r42119268>
> :
>
> > @@ -115,7 +117,8 @@ func NewProxyConfig() *ProxyServerConfig {
> >  		HealthzBindAddress: net.ParseIP("127.0.0.1"),
> >  		OOMScoreAdj:        qos.KubeProxyOOMScoreAdj,
> >  		ResourceContainer:  "/kube-proxy",
> > -		SyncPeriod:         30 * time.Second,
> > +		IptablesSyncPeriod: 30 * time.Second,
> > +		ConfigResyncPeriod: 10 * time.Minute,
>
> Would it be possible to randomize this number to avoid all KubeProxies
> resync at the same time?
> Can we e.g. change the flag to e.g. MinConfigSyncPeriod and make this
> value to be random from: [MinConfigSyncPeriod, 2*MinConfigSyncPeriod]
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15682/files#r42119268>.
>
>